test/helpers/types/types.go

```diff
@@ -195,7 +195,21 @@ type ViaQCommon struct {
 	// * dateOptionalTime
 	//
 	// example: `2015-01-24 14:06:05.071000000 Z`
-	Timestamp time.Time `json:"@timestamp,omitempty"`
+	TimestampLegacy time.Time `json:"@timestamp,omitempty"`
+
+	// A UTC value that marks when the log payload was created.
+	//
+	// Value derived from legacy `@timestamp` for forward compatibility.
+	//
+	// format:
+	//
+	// * yyyy-MM-dd HH:mm:ss,SSSZ
+	// * yyyy-MM-dd'T'HH:mm:ss.SSSSSSZ
+	// * yyyy-MM-dd'T'HH:mm:ssZ
+	// * dateOptionalTime
+	//
+	// example: `2015-01-24 14:06:05.071000000 Z`
```

Comment:
Shouldn't this be:

```suggestion
	// example: `2015-01-24T14:06:05.071000000Z`
```

Reply:
Quite possibly, I just copied the current.   Let me check into it.